packages/back-end/src/services/auth/index.ts

```diff
@@ -154,6 +155,31 @@ export async function processJWT(
           return;
         }
 
+        const memberRecord = req.organization.members.find(
+          (m) => m.id === req.userId
+        );
+        if (memberRecord) {
```

Comment:
Could you optimize this by passing a constraint in the `find`? Mongo seems to support searching for date ranges.

Reply:
Ah, this is an array `find` method and not a mongoose method. The `req.org.members` list is typically populated at this point with all org members (see preceding lines).